modules/consentManagementUsp.js

```diff
@@ -298,18 +299,21 @@ export function resetConsentData() {
  */
 export function setConsentConfig(config) {
   config = config && config.usp;
-  if (!config || typeof config !== 'object') {
-    logWarn('consentManagement.usp config not defined, exiting usp consent manager');
+  if (config && config.disable === true) {
```

Comment:
this allows a publisher to restore the old behavior with configuration.

Reply:
I don't think this is equivalent - it does not mean "disable" but "ignore". Let's say I do

```
pbjs.setConfig({
   consentManagement: {
      gpdr: {...}
   }
});
pbjs.setConfig({
  consentManagement: {
    usp: {
       disable: true
    }
  }
});
```

the first call would activate USP with the default values, the second would log a warning but otherwise do nothing - USP would still be enabled.

You could `resetConsentData` (and remove the `requestBidsHook`), but I can't think of another example where we ask for explicit *de*activation. To me it seems cleaner if we either require explicit activation (maybe that means accepting `usp: {}` to indicate the default), or just make it clear that if you install the module, it's always active.

Reply:
if what you say is true, how is this test passing

it('should not run if disable is true', function () {
        setConsentConfig({usp: {disable: true}});
        expect(consentAPI).to.be.undefined;
        expect(consentTimeout).to.be.undefined;
        sinon.assert.callCount(utils.logWarn, 1);
      });